src/core/qgsogcutils.cpp

```diff
@@ -1666,6 +1666,10 @@ static int binaryOperatorFromTagName( const QString& tagName )
 
 static QString binaryOperatorToTagName( QgsExpression::BinaryOperator op )
 {
+  if ( op == QgsExpression::boILike )
+  {
+    return "PropertyIsLike";
```

Comment:
Just to be sure, this should be mapped to *case insensitive* ilike and not like?

    'Banana' ILIKE 'b%'
    'Banana' LIKE 'B%'